openwisp_utils/utils.py

```diff
@@ -29,6 +29,48 @@ def register_menu_items(items, name_menu='OPENWISP_DEFAULT_ADMIN_MENU_ITEMS'):
         current_menu += items
 
 
+def register_default_menu_group_item(items):
+    menu_group_name = 'OPENWISP_DEFAULT_ADMIN_MENU_GROUPS'
+    if not hasattr(settings, menu_group_name):
+        default_group = OrderedDict()
+        default_group['Default'] = {'items': items}
+        setattr(settings, menu_group_name, default_group)
+    else:
+        default_group = getattr(settings, menu_group_name)
+        default_group['Default']['items'] += items
+
+
+def register_menu_groups(
+    menu_groups, menu_group_name='OPENWISP_ADMIN_MENU_GROUPS', preference=[]
+):
+    if not hasattr(settings, menu_group_name):
+        menu = OrderedDict()
+        setattr(settings, menu_group_name, menu)
+    else:
+        menu = getattr(settings, menu_group_name)
+    group_map = {}
+    for group in menu_groups:
+        if menu.get(group['name'], None) is not None:
+            # menu name already present
+            raise Exception(
```

Comment:
Can we make it `ArguementException` instead of `Exception`, I think that would more accurately represent the error.